src/components/Rating/Rating.st.css

```diff
@@ -0,0 +1,97 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-8;
+    DefaultEmptyColor: color-3;
+    DisabledColor: color-3;
+    IconGap: 3px;
+    IconLargeGap: 11px;
+    IconLargeSize: 30px;
+}
+
+:vars {
+    /* Overrides */
+    IconColor: --overridable;
+    IconEmptyColor: --overridable;
+}
+
+.root {
+    -st-states: mobile;
+
+    overflow: hidden;
+}
+
+.root:hover .icon:not(:disabled) path {
+    stroke: color(fallback(value(IconColor), value(DefaultColor)));
+}
+
+.iconList {
+    display: flex;
+    align-items: center;
+    flex-direction: row-reverse;
+
+    float: left;
+}
+
+.icon {
+    -st-states: checked, disabled, error, mode( enum(edit, preview) ), iconSize( enum(large) );
```

Comment:
It's better to add all the `states` in the `root` selector, and then do something like this:
```
.root:disabled .icon {
...
}
``` 
using states in other selectors can cause unwanted bugs.